src/main/markdown/doc/latest/RefJreEmulation.md

```diff
@@ -6,15 +6,73 @@ The list below shows the set of JRE packages, types and methods that GWT can tra
 Note that in some cases, only a subset of methods is supported for a given type.
 
 <ol class="toc" id="pageToc">
+  <li><a href="#Package_java_beans">java.beans</a></li>
```

Comment:
The entries are reordered here due to https://gwt-review.googlesource.com/c/gwt/+/19840